app/Providers/AppServiceProvider.php

```diff
@@ -49,6 +49,26 @@ public function registerServicesFromAura()
             return service('repository.message');
         });
 
+        $this->app->singleton(\Ushahidi\Core\Entity\ContactRepository::class, function ($app) {
```

Comment:
Not sure what the sustainable path is for this going forward, but for now this works